### PR TITLE
Unique prices in tooltip

### PR DESCRIPTION
**Pull request by makhotkin**
Select a table in poe.ninja and press Crtl+v into unique_prices.txt, then see prices in game

Very useful to clean stash tabs

### CODE REVIEW

File: src/Hud/AdvancedTooltip/AdvancedTooltipPlugin.cs [hunk 1]
```diff
@@ -28,45 +29,54 @@ public class AdvancedTooltipPlugin : Plugin<AdvancedTooltipSettings>
         private readonly SettingsHub settingsHub;
         private Entity itemEntity;
 		private int iLvl;
+		private string uniqueName;
         private List<ModValue> mods = new List<ModValue>();
+		private ItemRarity rarity;
 
-        public AdvancedTooltipPlugin(GameController gameController, Graphics graphics, AdvancedTooltipSettings settings, SettingsHub settingsHub)
+		Dictionary<string, string> uniquePrices = LoadUniquePrices();
```
Comment (smad2005) [inline src/Hud/AdvancedTooltip/AdvancedTooltipPlugin.cs]: `Lazy<Dictionary<string, string>> uniquePrices  = new Lazy<Dictionary<string, string>>(LoadUniquePrices);`

**Comment by uLow** [comment]
I'm using poe.ninja api to receive current prices. Otherwise you have to update unique_prices.txt manually quite often. 

API example: http://cdn.poe.ninja/api/Data/GetUniqueWeaponOverview?league=Legacy

**Comment by uLow** [comment]
And why are you trying to put it here instead of Plugin?

**Comment by TehCheat** [comment]
Yes, this should be a plugin rather than built into code.

**Comment by uLow** [comment]
Btw is it planning to make x32 plugins too? Most of PoEHUD users are using 32 bit version (it's more stable and works faster even on good PCs).

**Comment by zaafar** [comment]
No.

**Comment by zaafar** [comment]
But if you want you can port the code from x64 to x32 branch and maintain it.

**Comment by uLow** [comment]
Is it that hard or there are pitfalls i should know about?

**Comment by uLow** [comment]
I could give it a try on weekend.

**Pull request closed without merging**